src/incubator/WheelPicker/index.tsx

```diff
@@ -78,6 +78,10 @@ export interface WheelPickerProps {
    */
   separatorsStyle?: ViewStyle;
   testID?: string;
+  /**
+   * Change the default (white) tint color of the fade view.
+   */
+  tintColor?: string;
```

Comment:
I'd change the prop name to something more explicit to avoid confusing with other elements. 
Something like `faderTintColor`

Reply:
Let me know if this is good enough